react/features/prejoin/components/preview/Preview.js

```diff
@@ -66,7 +67,6 @@ function Preview(props: Props) {
  */
 function mapStateToProps(state) {
     return {
-        name: getPrejoinName(state),
```

Comment:
remove this from props? or is it now passed explicitly?

Reply:
Yes, this is now passed explicitly.